### PR TITLE
[Improvement][dolphinscheduler-api] Add a switch for Swagger #10332

### LINKED ISSUE #10332: [Improvement][dolphinscheduler-api] Add a switch for Swagger
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description
#10332 
Swagger has unauthorized access, exposing too much sensitive information

### Use case

Add a switch for Swagger
Swagger has unauthorized access, exposing too much sensitive information

### Related issues

None

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!
- #10333 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kangyuanjia**
## Purpose of the pull request
Fix #10332 
Swagger has unauthorized access, exposing too much sensitive information

## Brief change log
    modified:   dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/SwaggerConfig.java
    modified:   dolphinscheduler-api/src/main/resources/swagger.properties

## Verify this pull request

This pull request is code cleanup without any test coverage.

*Yes*

This pull request is already covered by existing tests, such as *(please describe tests)*.

*Yes*

This change added tests and can be verified as follows:
*None*

### CODE REVIEW

File: dolphinscheduler-api/src/main/resources/swagger.properties [hunk 1]
```diff
@@ -16,3 +16,6 @@
 #
 
 springfox.documentation.swagger.v2.path=/api-docs
+
+# swagger-ui turn off:false turn on:true
+swagger-enable=true
```
Comment (SbloodyS) [inline dolphinscheduler-api/src/main/resources/swagger.properties:21]: That's great. Please also add this to our docs.
docs/docs/zh/guide/open-api.md
docs/docs/en/guide/open-api.md

BTW. Please associate the corresponding issue to this PR.

Comment (caishunfeng) [reply dolphinscheduler-api/src/main/resources/swagger.properties:21]: Hi @kangyuanjia thanks for your submit and please update the docs, thanks.

Comment (davidzollo) [reply dolphinscheduler-api/src/main/resources/swagger.properties:21]: > Hi @kangyuanjia thanks for your submit and please update the docs, thanks.

@kangyuanjia any progress?

**Comment by caishunfeng** [comment]
Close this pr due to longtime no update. Feel free to reopen it.

**Pull request closed without merging**